backend/ticketvise/models/inbox.py

```diff
@@ -51,7 +51,8 @@ class Inbox(models.Model):
                         * **users** -- Set of :class:`User` s belonging to the inbox.
     """
 
-    code = models.CharField(max_length=50, unique=True)
+    lti_context_label = models.CharField(max_length=255, null=True, blank=True)
+    lti_context_id = models.CharField(max_length=255, null=True, blank=True)
```

Comment:
Moet hier niet `unique=True` bij? Wat is nu de primary key van een inbox?

Reply:
Dat is nu de unique_together = ("lti_context_label", "lti_context_id") op regel 95.